crates/cac_client/Cargo.toml

```diff
@@ -19,6 +19,8 @@ serde_json = { workspace = true }
 strum = { workspace = true }
 strum_macros = { workspace = true }
 tokio = { version = "1.29.1", features = ["full"] }
+mini-moka = { version = "0.10.3" }
+itertools = "0.10.5"
```

Comment:
`context-aware-config` crate is also using this, can you please mark both as workspace and move this version definition to the common one